Description: Avoid build dependency on textlive-fonts-extra
 The tex sources seem to require this universe package to use
 DejaVuSansMono fonts. Replacing this with simple courier so
 we can drop the build-dependency.
 The resulting pdf documents seem to be readable. Only the
 commandline examples look a little smaller to me.
Forwarded: No
Author: Stefan Bader <stefan.bader@canonical.com>
Last-Update: 2015-12-01

--- a/doc/guides/conf.py
+++ b/doc/guides/conf.py
@@ -59,7 +59,7 @@
 # Latex directives to be included directly in the latex/pdf docs.
 latex_preamble = r"""
 \usepackage[utf8]{inputenc}
-\usepackage{DejaVuSansMono}
+\usepackage{courier}
 \usepackage[T1]{fontenc}
 \usepackage{helvet}
 \renewcommand{\familydefault}{\sfdefault}
